app/mbm/templates/mbm/base.html

```diff
@@ -41,6 +41,9 @@
 
     <!-- Override this block to add extra stylesheets -->
     {% block extra_css %}{% endblock %}
+
+    <!-- Override this block to add scripts which belong in the <head> -->
+    {% block head_js%}{% endblock %}
```

Comment:
**[Question, non-blocking]** Is there a reason to include this in the `<head>` where it has the potential to block rendering, as opposed to in the `<body>` after the HTML content? It seems like the script is set up to run async so I doubt it matters, but I'm curious!

Reply:
It's what they recommend in matomo, so I can only speculate. My guess is they want it to run before other scripts?